config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
I agree the wording would need to change regardless, since in an app context "refresh the page" sounds odd (there is no "page", just the app).